### PR TITLE
Update Gen AI Notebook

### PR DESCRIPTION
**Pull request by drbeh**
`num_inference_steps` (which is default to 1000) now can be set to 1 for CI tests speedup.

### CODE REVIEW

File: notebooks/Medical Image Generation with Generative AI.ipynb [hunk 1]
```diff
@@ -89,7 +89,10 @@
     "cs_bucket = \"<bucket/container name to push the experiment job data to>\"  # Please fill it with the actual bucket name\n",
     "\n",
     "# Job configuration\n",
-    "timeout = 3600                                                            # Time (in seconds) to wait for a job to be completed"
+    "timeout = 3600                                                            # Time (in seconds) to wait for a job to be completed\n",
+    "\n",
+    "# Please leave the following advanced parameter as None\n",
```
Comment (mingxin-zheng) [inline notebooks/Medical Image Generation with Generative AI.ipynb:94]: This is a bit confusing. How about saying: `None` will use the default value (N) here?

File: notebooks/Medical Image Generation with Generative AI.ipynb [hunk 2]
```diff
@@ -215,9 +218,10 @@
     "data = {\n",
     "    \"action\": \"generate\",\n",
     "    \"specs\": {\n",
-    "        \"num_output_samples\": 2,\n",
+    "        \"num_output_samples\": 1,\n",
```
Comment (mingxin-zheng) [inline notebooks/Medical Image Generation with Generative AI.ipynb:221]: Can we have inline comments about the specs used for Gen2D? Thanks!

**Review by mingxin-zheng — APPROVED** [review]

File: notebooks/Medical Image Generation with Generative AI.ipynb (hunk 1 above)
Comment (drbeh) [reply notebooks/Medical Image Generation with Generative AI.ipynb]: update it. thanks

File: notebooks/Medical Image Generation with Generative AI.ipynb (hunk 2 above)
Comment (drbeh) [reply notebooks/Medical Image Generation with Generative AI.ipynb]: Added the comment for each spec of GenAI. Thanks

**Pull request merged**